application/ui/src/app/pages/matching-group/matching-group.page.ts

```diff
@@ -115,10 +197,12 @@ export class MatchingGroupPage implements OnInit {
 
   clearGroup1() {
     this.selectComponentGroup1.clear();
+    this.selectComponentGroup1.close();
```

Comment:
are you sure here when we close you can no more access the component and you have console error, please check and remove if we have those errors in the console